apps/glusterfs/dbcommon.go

```diff
@@ -33,6 +33,30 @@ type Db struct {
 	PendingOperations map[string]PendingOperationEntry `json:"pendingoperations"`
 }
 
+//DbBucketCheckResponse ... is summary of check on a db bucket.
+type DbBucketCheckResponse struct {
+	Total           int      `json:"total"`
+	Pending         int      `json:"pending"`
+	Ok              int      `json:"ok"`
+	NotOk           int      `json:"notok"`
+	Inconsistencies []string `json:"inconsistencies"`
+}
+
+//DbCheckResponse ... is the output of db check. It lists a summary of db state
+//and inconsistencies related to each bucket, if any.
+type DbCheckResponse struct {
+	Clusters             DbBucketCheckResponse `json:"clusters"`
+	Volumes              DbBucketCheckResponse `json:"volumes"`
+	Bricks               DbBucketCheckResponse `json:"bricks"`
+	Nodes                DbBucketCheckResponse `json:"nodes"`
+	Devices              DbBucketCheckResponse `json:"devices"`
+	BlockVolumes         DbBucketCheckResponse `json:"blockvolumes"`
+	DbAttributes         DbBucketCheckResponse `json:"dbattributes"`
+	PendingOperations    DbBucketCheckResponse `json:"pendingoperations"`
+	TotalInconsistencies int                   `json:"totalinconsistencies"`
+	Inconsistent         bool                  `json:"inconsistent"`
```

Comment:
This parameter seems redundant, given that we can just check if TotalInconsistencies is non-zero.

Reply:
done